src/mono/netcore/nuget/Microsoft.NET.Runtime.MonoAOTCompiler.Task/Sdk/Sdk.props

```diff
@@ -0,0 +1,22 @@
+<!--
+***********************************************************************************************
+Sdk.props
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0">
+
+  <PropertyGroup>
+    <_AOTTasksDirectoryRoot Condition=" '$(_ILLinkTasksDirectoryRoot)' == '' ">$(MSBuildThisFileDirectory)../tools/</_AOTTasksDirectoryRoot>
+    <_AOTTasksTFM>net5.0</_AOTTasksTFM>
```

Comment:
I don't know if the TFM should be hardcoded or not.  It should follow whatever `$(NetCoreAppToolCurrent)` is, but it's not a value when a developer would use the package.  

If we leave it hard coded, then we would need to change it when `$(NetCoreAppToolCurrent)` bumps.  Chances are pretty good we would not remember and bad things will happen.

Reply:
Is it necessary to have this file? Could the user importing the package create their own UsingTask as how it is done here? https://github.com/dotnet/samples/blob/1bb5dc93e2ce9edafddbee8c9dfd25cd31dafdf1/core/mono-samples/wasm/WasmSampleApp.csproj#L11-L16

Reply:
Good question.  My preference would be to have all you need in the package (keep it).  That way the xamarin sdk's and whoever else can just use it.

Reply:
Why is TFM needed here?

Reply:
`_AOTTasksTFM` is included to resemble the path to `MonoAOTCompiler.dll` in the NuPkg to make the task readily available to those using this NuPkg via `<UsingTask TaskName="MonoAOTCompiler" AssemblyFile="$(AOTTasksAssembly)" />` at the end of this file.

`MonoAOTCompiler.dll` is located at `tools\$(NetCoreAppToolCurrent)\` in the NuPkg by 
`<PackageFile Include="@(_AOTCompilerTaskFile)" TargetPath="tools\$(NetCoreAppToolCurrent)\" />` in `Microsoft.NET.Runtime.MonoAOTCompiler.Task.pkgproj`.

Reply:
Instead, going to have the user import the NuPkg and standup the MonoAOTCompiler task with their own UsingTask, as detailed in the `README.md`